app/components/mrl-search-result.js

```diff
@@ -0,0 +1,9 @@
+import Ember from 'ember';
+
+export default Ember.Component.extend({
+  actions: {
+    toggleBody: function() {
+      this.toggleProperty('isShowingBody');
+    }
+  }
+});
```

Comment:
:+1: 

I actually would like to do this for all search results. I have it noted in my personal notes to do this as a refactoring at some point. I'm glad you came to the same conclusion.

Ideally we'd find a way to DRY up the `toggleBody` action, but we for the time-being this is fine.